<ui version="4.0" >
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>519</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSimulation" >
    <property name="title" >
     <string>Simulation</string>
    </property>
    <addaction name="actionStart" />
    <addaction name="actionStop" />
    <addaction name="actionRestart" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuViews" >
    <property name="title" >
     <string>Views</string>
    </property>
    <addaction name="actionNew_Viewport" />
   </widget>
   <addaction name="menuSimulation" />
   <addaction name="menuViews" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionStart" >
   <property name="text" >
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="text" >
    <string>Stop</string>
   </property>
  </action>
  <action name="actionRestart" >
   <property name="text" >
    <string>Restart</string>
   </property>
  </action>
  <action name="actionNew_Viewport" >
   <property name="text" >
    <string>New Viewport</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
